docs: readme update (#670)
tidev/pulsar-titanium: README.md

# Titanium SDK Package for Pulsar [![Build Status](https://travis-ci.org/appcelerator/atom-appcelerator-titanium.svg?branch=master)](https://travis-ci.org/appcelerator/atom-appcelerator-titanium)

Titanium SDK build tools and UI for the [Pulsar text editor](https://pulsar-edit.dev/) (formally Atom).

<img src="https://user-images.githubusercontent.com/10667698/42400977-8e552cde-8141-11e8-8696-1380eb9dd581.png" width="900px" alt="Titanium SDK for Pulsar screenshot" />

## Getting Started

* Pulsar is required.

> * Install `pulsar-titanium` package.

Reviewer: Package name is just `titanium`, I think we do still need some extra packages but I'll need to check what else is needed

```suggestion
* Install `titanium` package.
```

### Using the Pulsar package manager

> Preferences/Settings > Install > Search for `pulsar-titanium`.

Reviewer: Package name is just titanium

```suggestion
Preferences/Settings > Install > Search for `titanium`.
```

### Using the CLI:

Use `ppm` (Pulsar package manager) or `apm` (Atom package manager):

```

> ppm install pulsar-titanium

Reviewer: ```suggestion
pulsar --package install pulsar-titanium
```

```

You can also use the repo URL to install the latest repo version:
```

> ppm install https://github.com/tidev/pulsar-titanium

Reviewer: I think they're leaning towards `pulsar --package` now over `ppm` (and I've had issues with it on Windows)
```suggestion
pulsar --package install https://github.com/tidev/pulsar-titanium
```

#### Build

> The build select button executes and provides selections of the build command. Depending on the target platform selected, the drop-down will display options for the `ti run` command:

Reviewer: ```suggestion
The build select button executes and provides selections of the build command. Depending on the target platform selected, the drop-down will display options for the `ti build` command:
```

<img src="https://user-images.githubusercontent.com/2697821/33169524-ed2b85ca-d03c-11e7-881e-4eb9e301649d.png" width="109px" alt="Build command select" />

* Run: Build and run the app on selected simulator/emulator or device
* Ad-Hoc: (iOS only) build and package the app for Ad-Hoc distribution using selected code signing
* Publish: Build and package the app for distribution through the App Store or Google Play using selected code signing or keystore credentials

> * Custom: Execute `ti run -d <project_dir>` with custom arguments

Reviewer: ```suggestion
* Custom: Execute `ti build -d <project_dir>` with custom arguments
```

#### Console

<img src="https://user-images.githubusercontent.com/2697821/33169526-ed531cf2-d03c-11e7-8d08-5657f691aca2.png" width="876px" alt="Build command select" />

> The console displays debug logging during the build and run. The log level is passed to the `appc run` comand so selecting the log level does not alter output from the currently executing command. There are options to automatically scroll the console to the latest message and to open it when a build is initiated.

Reviewer: ```suggestion
The console displays debug logging during the build and run. The log level is passed to the `ti build` command so selecting the log level does not alter output from the currently executing command. There are options to automatically scroll the console to the latest message and to open it when a build is initiated.
```
